modules/modelSetup/mixin/ModelSetupDebugMixin.py

```diff
@@ -39,31 +40,26 @@ def _save_text(self, text: str, directory: str, name: str, step: int):
         with open(path, "w") as f:
             f.write(text)
 
-    def _decode_tokens(self, tokens:Tensor, tokenizer):
+    def _decode_tokens(self, tokens: Tensor, tokenizer):
         return tokenizer.decode(
             token_ids=tokens[0],
             skip_special_tokens=True,
             clean_up_tokenization_spaces=True,
         )
 
-    # Decodes 4-channel latent to 3-channel RGB - technique appropriated from 
+    # Decodes 4-channel latent to 3-channel RGB - technique appropriated from
     # https://huggingface.co/blog/TimothyAlexisVass/explaining-the-sdxl-latent-space
     # Uses linear approximation based on first three channels of latent image (luminance, cyan/red, lime/purple)
     def _project_latent_to_image_sdxl(self, latent_tensor: Tensor):
-        weights = (
-            (60, -60, 25, -70),
-            (60, -5, 15, -50),
-            (60, 10, -5, -35)
-        )
+        weights = ((60, -60, 25, -70), (60, -5, 15, -50), (60, 10, -5, -35))
```

Comment:
Another case where manual line breaks make sense. The previous formatting made it more clear that this is a matrix